src/main/webapp/pages/projects/settings/pages/groups.html

```diff
@@ -28,7 +28,7 @@
             <div class="btn-group btn-group-sm">
                 <button id="add-members-button" class="btn btn-default"
                         type="button" data-toggle="modal" data-target="#addMemberModal"
-                        th:if="${isAdmin} or ${isOwner}" href="#addMembersLink"
+                        th:if="${isAdmin} or ${isOwner} or ${manageMembers}" href="#addMembersLink"
```

Comment:
You can just drop this to `${isAdmin} or ${manageMembers}`.  manageMembers (from `ManageLocalProjectSettingsPermission`) is kinda a subset of `isOwner`.  This would be the same as `pages/projects/settings/pages/members.html`

Reply:
Fixed in cd7d365fb9a215249c0dd68c14fb08ba6a8cc684